src/main/kotlin/domain/usecase/SuggestClothesUseCase.kt

```
@@ -0,0 +1,27 @@
+package org.damascus.domain.usecase
+
+import org.damascus.data.weather.dto.CurrentWeather
+import org.damascus.domain.model.Cloth
+import org.damascus.domain.model.ClothType
+import org.damascus.domain.repository.ClothesRepository
+
+class SuggestClothesUseCase(
+    private val clothesRepository: ClothesRepository
+) {
+    operator fun invoke(currentWeather: CurrentWeather): List<Cloth> {
+        val temp = currentWeather.temperature
+
+        if (temp.isNaN()) return emptyList()
```

Should throw an exception instead of an empty list.